web/src/main/webapp/my-app/layout/services.js

```diff
@@ -42,6 +42,7 @@ define(['angular', 'jquery'], function(angular, $) {
         ************************/
 
         if (APP_FLAGS.useNewLayout) {
+          console.log("using new layout");
```

Comment:
if possible it can be good to avoid extra logs, or to wrap logs with https://github.com/visionmedia/debug#readme to allow easier management of what is logging and what levels of logging are shown.

```suggestion
```